### PR TITLE
updated project details and removed duplicate project

### PR DESCRIPTION
**Pull request by sukumarc**
Removed duplicate projects and added my idea !!!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -113,22 +113,13 @@ We'll organise into different streams here. One kind request for motivation: Ple
...
-* Library Managemnet App
-* Authorization Server
-* Chatting App
-* eWallet App
-* Online Store App
-
-## Back End Ideas
-
-### Spring Stack
-
```
Comment (praveenscience) [inline README.md:125]: Is it intentional deleting all these?

**Review by praveenscience — CHANGES_REQUESTED** [review]
Can you check?

File: README.md (hunk 1 above)
Comment (sukumarc) [reply README.md:125]: Yes, I have deleted the in intentionally for duplicate names. Spring stack is present 2 times.

**Review by praveenscience — APPROVED** [review]
👍

**Comment by praveenscience** [comment]
Thanks for your contribution. Optional Request: [Subscribe to my **YouTube Channel**](https://www.youtube.com/praveenscience?sub_confirmation=1), if you haven't already subscribed and share it with your friends too.

**Pull request merged**